### ISSUE TITLE
“Minimap-slider-horizontal bar” is not visible after applying high contrast theme under Query Playground blade.

### ISSUE THREAD

**Issue by Amritakumari12** [microsoft/monaco-editor#2375]
Test Environment:
OS Version: 20H2 (OS Build: 19042.388)
Browser: Edge chromium DEV Build (Version 85.0.552.1), Firefox(78.0.2 (32-bit)

Repro Steps:​
1- Launch Azure Cosmos DB Website.
2- Home page will open.
3-  Navigate to "Top" tab appearing in Home page.
4. Navigate through the controls appearing under "Query", "Results" and "Sample Doc" modules.
5. Repet step 4 for "In & Between" and "Projection" tabs.
6. Switch to high contrast black mode.
7. Verify the issue.

Actual Result:
“Minimap-slider-horizontal bar” is not visible after applying high contrast theme under Query Playground blade. 

Expected Result:
“Minimap-slider-horizontal bar” should visible after applying high contrast theme under Query Playground blade.

Note:
Issue does not exists in high contrast white mode.

![Top tab-minimap-slider-horizontal bar is not visible after applying high contrast theme](https://user-images.githubusercontent.com/40196203/105136405-95c4a500-5b17-11eb-9a36-c4b8a01e5d9e.png)
